docs/config.toml

```diff
@@ -115,13 +115,13 @@ version = "0.5.8"
 
 # A link to latest version of the docs. Used in the "version-banner" partial to
 # point people to the main doc site.
-url_latest_version = "https://rancher-sandbox.github.io/cos-toolkit/"
+url_latest_version = "https://rancher.github.io/elemental-toolkit/"
 
 # Repository configuration (URLs for in-page links to opening issues and suggesting changes)
-github_repo = "https://github.com/rancher-sandbox/cOS-toolkit"
+github_repo = "github.com/rancher/elemental-toolkit"
```

Comment:
gotta try this in a preview, not sure if it wants the URI scheme

Reply:
Ouch, my fault 🤦🏼‍♂️